test/reporter-html/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="UTF-8">
 	<title>QUnit HTML Reporter Test Suite</title>
-	<link rel="stylesheet" href="../dist/qunit.css">
+	<link rel="stylesheet" href="../../dist/qunit.css">
```

Comment:
Do you want me to remove that? I couldn't run the QUnit HTML Reporter suite without that change.